From a5d29b53bbbc00d57540be09d954cf54a3278a00 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Fri, 4 Jan 2019 11:50:09 +0100
Subject: [PATCH] configure.ac: restore normal behavior for CC, LD and AR

The configure.ac should not override those environment variables, as
it breaks the standard behavior of configure scripts, making the
integration of musdk in build systems that rely on standard autotools
behavior more complicated for no reason.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
---
 configure.ac | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 38685cc..02218de 100644
--- a/configure.ac
+++ b/configure.ac
@@ -261,9 +261,6 @@ CFLAGS+="-pthread "
 CFLAGS+="$MUSDK_CFLAGS $SAM_CFLAGS $PP2_CFLAGS"
 LDFLAGS+="$SAM_LDFLAGS"
 LIBS+="$SAM_LIBS"
-CC=${CROSS_COMPILE}gcc
-LD=${CROSS_COMPILE}ld
-AR=${CROSS_COMPILE}ar
 
 # define LIBTOOL (for libraries)
 LT_INIT
-- 
2.20.1

